### PR TITLE
update http functional test to test using password

### PR DESCRIPTION
**Pull request by jcantrill**
### Description
This PR:
Modifies the functional HTTP test to test using username and password

### CODE REVIEW

File: test/functional/outputs/http/forward_to_http_test.go [hunk 1]
```diff
@@ -76,10 +77,33 @@ var _ = Describe("[Functional][Outputs][Http] Functional tests", func() {
 		Expect(logs[0].Message).To(Equal(message))
 		Expect(logs[0].Kubernetes.Labels).To(HaveKey(MatchRegexp("^([a-zA-Z0-9_]*)$")))
 		Expect(logs[0].Kubernetes.Labels).To(HaveKey(MatchRegexp("foo")))
+		Fail("")
 	},
-		Entry("should send message over http to vector", func(f *functional.CollectorFunctionalFramework) runtime.PodBuilderVisitor {
+		FEntry("should send message over http to vector", func(f *functional.CollectorFunctionalFramework) runtime.PodBuilderVisitor {
```
Comment (Clee2691) [inline test/functional/outputs/http/forward_to_http_test.go:82]: Need to remove `F` from this test entry.

**Comment by cahartma** [comment]
needs a rebase

**Review by cahartma — COMMENTED** [review]
/lgtm

**Review by cahartma — COMMENTED** [review]
/lgtm

**Pull request merged**